Server/core/CDynamicLibraryInterface.h

```diff
@@ -0,0 +1,74 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        core/CDynamicLibrary.h
+ *  PURPOSE:     Dynamic library handling class
+ *
+ *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *
+ *****************************************************************************/
+
+#pragma once
+
+#ifdef _WIN32
+    #ifndef WIN32_LEAN_AND_MEAN
+        #define WIN32_LEAN_AND_MEAN
+    #endif
+    #include <windows.h>
+#else
+    #include <dlfcn.h>
+#endif
+
+#include <cstdlib>
+#include <cstring>
+#include <cstdarg>
+#include <cstdio>
+
+using FuncPtr_t = void (*)();
+
+class CDynamicLibraryInterface
```

Comment:
What's this?

Reply:
Replacement for `typedef`. Its just a C++ version of function alias